richeditor-compose/src/commonMain/kotlin/com/mohamedrejeb/richeditor/model/RichTextState.kt

```diff
@@ -304,20 +304,37 @@ public class RichTextState internal constructor(
         textRange: TextRange,
         text: String
     ) {
-        require(textRange.min >= 0) {
```

Comment:
I really prefer to force the user to provide a correct textRange.
With this new check we will get some unexpected results, let's say you enter (-10, -5) range then you well get an error saying `"The start index (0) must be less than or equal to the end index (-5)."` but the start index was -10 not 0.
Let's keep it the same.